IdentityCore/src/webview/systemWebview/MSIDSystemWebviewController.m

```diff
@@ -26,19 +26,91 @@
 //------------------------------------------------------------------------------
 
 #import "MSIDSystemWebviewController.h"
+#import "MSIDSFAuthenticationSession.h"
+#import "MSIDSafariViewController.h"
 
 @implementation MSIDSystemWebviewController
+{
+    id<MSIDRequestContext> _context;
+    MSIDWebUICompletionHandler _completionHandler;
+
+#ifdef __IPHONE_11_0
+    MSIDSFAuthenticationSession *_authSession;
```

Comment:
nit: this is not set anywhere

Reply:
not a nit, huge mistake :p this should've been set 👍